test/commands/compareReports-test.js

```diff
@@ -1,7 +1,13 @@
+import path from 'path';
+
+import { createServer } from 'http-server';
+
 import compareReports from '../../src/commands/compareReports';
 import * as defaultConfig from '../../src/DEFAULTS';
 import makeRequest from '../../src/makeRequest';
 
+jest.setTimeout(60000);
```

Comment:
Is this because creating the http server is slow? Could these timeout increases be scoped to just the tests that need it (`it` takes a third argument for this)? Alternatively, could we use jest mock to prevent the need for an actual http server in these tests?

Reply:
Didn't know about `it`'s third argument, thanks! Using a proper server helps surface a few issues that come as a result of running a lot in parallell, so I think I'll keep it. 

The increased timeout is to make sure that the new "giant" test doesn't time out.